### ISSUE TITLE
Movie Support

### ISSUE THREAD

**Issue by Geoo15** [IanMitchell/deschtimes#35]
Would be nice to have a slider to switch from a series to a movie. It would remove options such as First Episode Number, Episodes in Season, First Episode Air Date, etc.

**Comment by IanMitchell** [comment]
Makes sense yeah! I don't think I'll be able to get to this anytime soon, but would definitely be open to PRs with support for it
